__test__/main.test.ts

```diff
@@ -18,9 +19,13 @@ describe("workflow test without input", () => {
 		await expect(linux.runLinux()).rejects.toThrow();
 	});
 
-	it("run macOS workflow without input", async () => {
+	it("run MacOS workflow without input", async () => {
```

Comment:
```suggestion
	it("run macOS workflow without input", async () => {
```
I see macOS being used in the README, let's keep it that way.